include/openssl/core_names.h

```diff
@@ -460,6 +463,7 @@ extern "C" {
 #define OSSL_SIGNATURE_PARAM_MGF1_PROPERTIES    \
     OSSL_PKEY_PARAM_MGF1_PROPERTIES
 #define OSSL_SIGNATURE_PARAM_DIGEST_SIZE        OSSL_PKEY_PARAM_DIGEST_SIZE
+#define OSSL_SIGNATURE_PARAM_NONCE_TYPE         "nonce_type"
```

Comment:
in the updated documentation, the new param string is stated to be `nonce-type`.

Should this be `nonce-type` rather than `nonce_type`?

Reply:
yeah we should change this... before it makes it into a actual release.